core/src/test/scala/integration/kafka/server/ScramServerStartupTest.scala

```diff
@@ -1,65 +0,0 @@
-/**
```

Comment:
Why did you delete this test? Can you add an explanation to the PR description?

Reply:
I moved it to SaslScramSslEndToEndAuthorizationTest.scala. Setting up SCRAM in this file was far more code than the original test and duplicating what was already in SaslScramSslEndToEndAuthorizationTest.scala. It was much cleaner to just move it.

Reply:
Updated PR description to explain the move.